specs/catalog-schema-registry.spec/src/main/scripts/io/aklivity/zilla/specs/catalog/schema/registry/schema/schema.registry.schema.patch.json

```diff
@@ -98,23 +98,44 @@
                 {
                     "title": "Credentials",
                     "type": "object",
-                    "properties":
-                    {
-                        "headers":
+                    "oneOf": [
                         {
-                            "title": "Headers",
-                            "type": "object",
-                            "properties":
-                            {
-                                "authorization":
-                                {
-                                    "type": "string"
+                            "properties": {
```

Comment:
@jfallows 

In current schema, we define the `authorization` under `headers` parameter explicitly, as we pass this token as header in our request. 
```
    options:
      credentials:
        headers:
          authorization: Basic dXNlcjpzZWNyZXQ=
```

In similar way, suggested change should also follow same pattern right? Because even though we Base encode username & password internally, it is still passed as `authorization` header. 

I suggest something like this: 
```
    options:
      credentials:
        headers:
          authorization: 
            username: ${{env.SCHEMA_REGISTRY_USERNAME}}
            password: ${{env.SCHEMA_REGISTRY_PASSWORD}}
```

any insights?